server/bin/gen-certs

```diff
@@ -47,19 +81,153 @@ if [ ! -d $UPSTREAM_CERTS_HOME ]; then
+
+#     echo
+#     echo "import $SERVER_CERT server cert into candlepin keystore"
+     # import pck7 version into keystore?
+#     sudo keytool -importcert -trustcacerts -v -v -v -keystore $KEYSTORE \
+#        -storepass $KEYSTORE_PASS  -file $SERVER_CERT  -alias "tomcat" \
+#        -keypass $KEY_PASS -storetype $STORETYPE
+    
+    
+    #echo "Adding the CA cert to tomcats keystore, and create a pkcs keystore"
+    #sudo openssl pkcs12 -export -in $CA_CERT -inkey $CA_KEY -out $KEYSTORE \
+    #sudo openssl pkcs12 -export -noiter -nomaciter -in $CA_CERT -inkey $CA_KEY -out $KEYSTORE \
+    #    -name "candlepin-ca" -CAfile $CA_CERT -caname root -chain 
+    #sudo openssl pkcs12 -export -in $SERVER_CERT -inkey $SERVER_KEY -out $KEYSTORE \
+    #    -CAfile $CA_CERT -name "tomcat" -certfile $CA_CERT  -password pass:password -chain -caname root
+    #sudo keytool -import -trustcacerts -alias "candlepin-ca" -file CAcert.crt -keystore $JAVA_HOME/lib/security/cacerts 
+
+
+    echo "Create a keystore for tomcat"
+    sudo keytool -genkey -keystore $KEYSTORE \
+        -dname "CN=$HOSTNAME , OU=Candlepin Server, O=candlepinproject.org" \
+        -keypass $KEY_PASS \
+        -storepass $KEYSTORE_PASS \
+        -keyalg RSA \
+        -alias "tomcat" -storetype $STORETYPE
+
+    if [ -n "$VERBOSE" ] ; then
+        echo
+        echo "CA CERT $CA_CERT"
+        sudo keytool -printcert -v -file $CA_CERT
+
+        echo
+        echo "apache SERVER CERT $SERVER_CERT"
+        sudo keytool -printcert -v -file $SERVER_CERT
+    fi
+
+    echo
+    echo "import $CA_CERT ca cert into tomcat keystore"
+    sudo keytool -importcert -v -v -v -keystore $KEYSTORE \
+        -storepass $KEYSTORE_PASS -file $CA_CERT  -alias $CANDLEPIN_CA_ALIAS \
+        -keypass $KEY_PASS -storetype $STORETYPE -trustcacerts \
+        -noprompt
+
+    if [ -n "$VERBOSE" ] ; then
+        echo
+        echo "keystore "
+        sudo keytool -list -v -keystore $KEYSTORE -storepass $KEYSTORE_PASS \
+            -storetype $STORETYPE
+    fi
+    #echo "convert server cert pkcs12 to pkcs7 for import to keystore"
+    # import again as an intmt cert?
+    # convert server-cert to pkc7 so we can import to candlepin keystore?
+    #   sudo openssl crl2pkcs7  -nocrl -certfile $SERVER_CERT \
+    #     -out $SERVER_CERT_P7 -certfile $CA_CERT_P7
+
+    echo
+    echo "Create a tomcat server cert signing request based on keystore"
+    sudo keytool -certreq -v -alias "tomcat" -keystore $KEYSTORE -storepass $KEYSTORE_PASS -file $TOMCAT_SERVER_CERT_CSR
+
+    echo
+    echo "Sign the tomcat server cert with the CA cert/key $CA_CERT/$CA_KEY"
+    sudo openssl x509 -req -in $TOMCAT_SERVER_CERT_CSR -CA $CA_CERT -CAkey $CA_KEY -CAcreateserial -out $TOMCAT_SERVER_CERT -days 365
+
+    echo
+    echo "Import the signed tomcat server cert $TOMCAT_SERVER_CERT into keystore $KEYSTORE"
+    #    sudo keytool -importcert -trustcacerts -v -v -v -keystore $KEYSTORE \
```

Comment:
Did `importcert` not work?  Creating the key and csr in keytool but still having to go through the PKCS12 to JKS dance makes the first part kind of pointless.